.github/workflows/ci.yml

```diff
@@ -17,14 +17,15 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v2
 
-      # Build the packages using the different feature flags
+      # Build the packages
```

Comment:
probably want to add clippy as a build step as well

Reply:
Implemented in https://github.com/autometrics-dev/autometrics-rs/pull/96